README.md

```diff
@@ -37,15 +38,21 @@ Koa, GraphQL and TypeScript template project with batteries included.
 
 ## Required Software
 
-- node >= 7.6.0
+- node >= 7.6.0 & npm / yarn
 
-# Install
+or
 
-`$ npm i` or `$ yarn`
+- Docker
+
+## Install
+
+`$ npm i` or `$ yarn` or `$ docker-compose up`
 
 ## Develop
 
-`$ npm start` or `$ yarn start`
+`$ npm start` or `$ yarn start` or `$ docker-compose up`
```

Comment:
By default `docker-compose up` does not re-build the image and you might end up running an old cached version. You need to add --build.

Reply:
Fair enough, will add a remark.